scripts/superdesk-authoring/views/article-edit.html

```diff
@@ -201,5 +201,11 @@
 
 <div class="field pull-left" ng-if="item.sign_off && schema.sign_off" order="{{schema.sign_off.order}}">
     <label translate>Sign-Off</label>
-    <div>{{item.sign_off}}</div>
+    <div>{{item.sign_off}}
+      <button class="btn btn-signOff" ng-click="changeSignOffEdit()" title="Unlock edit Sign-Off" ng-if="canEditSignOff && _editable">
+  		  <i class="icon-lock" ng-hide="editSignOff"></i>

```

Comment:
Maybe you can add <i `ng-class="{'icon-lock': !editSignOff, 'icon-trash': editSignOff}"></i>`